howso/hyperparameters.amlg

```diff
@@ -795,23 +790,23 @@
 	)
 
 	;sets the model to auto-analyze by tracking its size and notifying the clients in train responses when it should be analyzed
-	;parameters:
-	; auto_analyze_enabled: flag, default is false. when true, returns when it's time for model to be analyzed again.
-	; analyze_threshold: optional, stores the threshold for the number of cases at which the model should be re-analyzed. default of 100.
-	; analyze_growth_factor: the factor by which to increase the analyze threshold everytime the model grows to the current threshold size
-	;						default of two orders of magnitude using the universal scaling factor e
-	;
-	; optionally, if parameters that are specified for analyze are passed in, they will be stored and used during auto-analysis
 	#set_auto_analyze_params
 	(declare
 		(assoc
 			;{type "boolean"}
+			;flag, default is false. when true, returns when it's time for model to be analyzed again.
 			auto_analyze_enabled (false)
 			;{type "number"}
+			;stores the threshold for the number of cases at which the model should be re-analyzed. default of 100.
 			analyze_threshold 100
 			;{type "number"}
+			;the factor by which to increase the analyze threshold everytime the model grows to the current threshold size
+			;	default of two orders of magnitude using the universal scaling factor e
 			analyze_growth_factor 7.389056
-
+			;{type "number"}
+			;the size of of the model at which to stop doing outo-analysis. Value of 0 means no limit.
+			;if parameters that are specified for analyze are passed in, they will be stored and used during auto-analysis
+			auto_analyze_limit_size 200000
```

Comment:
this parameter doesn't exist so it and its comments should be removed

Reply:
Removed in 1632325